### ISSUE TITLE
User Feedback Requested : Increment an property that doesnt exist 

### ISSUE THREAD

**Issue by sharan779** [AzureCosmosDB/PatchPrivatePreviewFeedbackGroup#10]
Do you think Increment op should be a blind increment operator, meaning that the user might want to increment a property that might not even exist. 

In this case, would you vote for having a default value of 0 or an extra optional parameter "ifNotExists" ? 
![image](https://user-images.githubusercontent.com/1634290/119621981-4e3d8500-be39-11eb-8518-486f29a6fbd7.png)

Let us know your comments ?

**Comment by WWeichselberger** [comment]
My opinion on this: Keep it simple!
- **_No_** optional parameter
- If the property does not exist, it is initialized with 0 and incremented from there.

This should be just fine for most use cases. Anything more complicated (init value != 0; require already existing property) could be done with [conditional patches](https://github.com/AzureCosmosDB/PatchPrivatePreviewFeedbackGroup#conditional-patch-syntax-based-on-filter-predicate).

**Comment by fmamic** [comment]
I agree with @WWeichselberger and there is similar functionality for MongoDB:
https://docs.mongodb.com/manual/reference/operator/update/inc/

![image](https://user-images.githubusercontent.com/9407124/120222535-fedfc600-c237-11eb-8a40-44556d6ce4b7.png)

**Comment by ryancrawcour** [comment]
+1 for the Keep it Simple, and Keep it familiar to other database systems where we can already do this.
